server/chat-plugins/blackjack.js

```diff
@@ -0,0 +1,643 @@
+/**
+ * Blackjack Game
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This allows users to play the classic blackjack card game.
+ * Credits: jd, panpawn
+ *
+ * @license MIT license
+ */
+'use strict';
+
+class Blackjack extends Rooms.RoomGame {
+	constructor(room, user, target) {
+		super(room);
+		this.room = room;
+
+		this.turnTimeoutMinutes = 1;
+		this.timerTickSeconds = 5;
+
+		this.blackjackNumber = 0;
+		this.createdBy = user.name;
+		this.startedBy = '';
+		this.allowRenames = true;
+
+		this.playerCap = 16;
+		this.minimumPlayers = 2;
+		this.playerScrollWheel = 4;
+		this.cardWidth = 50;
+		this.cardHeight = 85;
+
+		this.spectators = Object.create(null);
+		this.dealer = new BlackjackDealer();
+
+		this.symbols = {
+			'♥': 'Hearts',
+			'♦': 'Diamonds',
+			'♣': 'Clubs',
+			'♠': 'Spades',
+		};
+		this.deck = new BlackjackDeck().shuffle();
+
+		this.id = this.room.id;
+		this.title = `Blackjack (${room.title})`;
```

Comment:
I don't believe any other games include the room title like this

Reply:
This is mostly so in the PS panel to the left, it'll say what room it's in. I can remove it if that isn't intended for that use-case though.